internal/connector/connector_test.go

```diff
@@ -0,0 +1,341 @@
+package connector_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"go.arcalot.io/assert"
+	"go.arcalot.io/log/v2"
+	"go.flow.arcalot.io/deployer"
+	"go.flow.arcalot.io/pluginsdk/atp"
+	"go.flow.arcalot.io/pluginsdk/schema"
+	pythondeployer "go.flow.arcalot.io/pythondeployer"
+	"go.flow.arcalot.io/pythondeployer/internal/config"
+	"go.flow.arcalot.io/pythondeployer/internal/connector"
+	"io"
+	"math/rand"
+	"os"
+	"os/exec"
+	"sync"
+	"testing"
+)
+
+func GetPythonPath() (string, error) {
+	python3Path, errPython3 := exec.LookPath("python3")
+	if errPython3 != nil {
+		pythonPath, errPython := exec.LookPath("python")
+		if errPython != nil {
+			return "", fmt.Errorf("error getting Python3 (%s) and python (%s)", errPython3, errPython)
+		}
+		return pythonPath, nil
+	}
+	return python3Path, nil
+}
+
+const examplePluginNickname string = "pythonuser"
+
+var inOutConfigGitPullIfNotPresent = `
+{
+	"workdir":"/tmp",
+	"modulePullPolicy":"IfNotPresent",
+    "pythonSemver": "3.0.0"
+}
+`
```

Comment:
Is the (lack of) indentation of line 41 significant?

Reply:
The multi-line strings do utilize all indentation present.

Reply:
The `"pythonSemver"` line is indented with four spaces while the preceding two lines are indented with a single hard-tab...I presume that all three should be using the same indentation...no?

Reply:
that odd indentation doesn't show up locally for me, so I don't know how to fix that right now

Reply:
This conversation isn't showing the updated code.